pkg/object/objects.go

```diff
@@ -22,10 +22,11 @@ import (
 	"bufio"
 	"bytes"
 	"fmt"
+	"io"
 	"sort"
 	"strings"
 
-	"github.com/ghodss/yaml"
+	ghodssyaml "github.com/ghodss/yaml"
```

Comment:
this is a good idea but we should stay consistent. i propose reverting this and starting a new PR to make this change throughout the project.